jsonnetfile.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
No reason to commit this `jsonnetfile.json`, since it doesn't contain any custom dependencies. Please don't add it to `.gitignore` in the repo though, since there may be a reason to add it in the future.

If having the untracked file bothers you, you can add it to the repo's `.git/info/exclude` or a global gitignore (configure a global gitignore with`git config --global core.excludesfile=/path/to/global/gitignore`)